app/Command/DCE.hs

```diff
@@ -350,9 +351,8 @@ dceCommand DCEOptions {..} = do
     -- we do not have access to data to regenerate extern files (they relay on
     -- more information than is present in `CoreFn.Module`).
     for_ mods $ \m -> lift $ do
-      let mn = P.runModuleName $ CoreFn.moduleName m
-      exts <- BSL.readFile (dceInputDir </> T.unpack mn </> "externs.json")
-      BSL.writeFile (dceOutputDir </> T.unpack mn </> "externs.json") exts
+      let mn = CoreFn.moduleName m
+      copyExterns mn "cbor" <|> copyExterns mn "json"
```

Comment:
This adds externs.cbor files to Zephyr output directory when used with PureScript <0.13.8 and I’ve no idea why. There’s no extraneous externs.json files when trying to copy them first with PureScript v0.13.8 🤔

Reply:
This was happening because I updated `zephyr` to depend on `purescript-0.13.8` so a dummy externs.cbor was always generated during codegen and wasn’t overridden when copying externs.json files generated by `purescript-0.13.6` or earlier.